test/cpp_tests_list_IECPU.txt

```diff
@@ -21,6 +21,7 @@ EncapsulateOp.AllocateNGInputTensors
 EncapsulateOp.AllocateNGOutputTensors
 OpByOpCapability.Backend
 MathOps.Sum
+MathOpsSumFixture.FullSet
```

Comment:
I just noticed that this wasn't in the `MathOps` namespace. Can you make sure it belongs to the `MathOps.Sum.*` namespace, even temporarily?

Reply:
I don't know of a way. First arg of TEST_F has to be a class. The MathOpsSumFixture class is specific for Sum related tests, so can't really name it MathOps.

Reply:
Ok, it's a little odd but I understand that it's just an example. We should use TEST_P for parameterized tests.